### PR TITLE
Update DF system scripts

### LINKED ISSUE #630: Update df-system scripts
There are some outdated scripts in df-systems, update them.

### PR DESCRIPTION
**Pull request by trizin**
Fixes #630

Changes proposed in this PR:

- Add staging scripts.
- Add missing update cloud script.

### CODE REVIEW

**Review by calina-c — APPROVED** [review]
LGTM but I would suggest getting a second review since I'm not yet familiar enough with this part.

**Review by trentmc — CHANGES_REQUESTED** [review]
Is the usage of the scripts documented somewhere? Eg from a README that's linked to the main README?

"It doesn't exist if it's not documented"

**Review by trentmc — APPROVED** [review]

**Pull request merged**
